lib/monitor.js

```diff
@@ -68,6 +68,15 @@ class Monitor {
         if (this.settings.wreck) {
             this._wreckHandler = (error, request, response, start, uri) => {
 
+                // Wreck 11+ emits response event with signature (err, {req, res, start, uri})
+                const isWreckVersion11OrHigher = (!response && typeof request === 'object');
```

Comment:
Thoughts on making this a proper `function` and checked `arguments.length` instead?